spec/support/downloads_helper.rb

```diff
@@ -26,7 +26,7 @@ def with_empty_downloads_folder
   private
 
   def downloaded_filenames
-    Dir[DownloadsHelper.path.join("*")]
+    Dir[DownloadsHelper.path.join("*")].reject { |f| File.directory?(f) }
   end
```

Comment:
For reviewers.
these changes to prevent this error:


```
Failures:


  1) enterprise fee summaries csv downloads smoke test for generation of report based on permissions when logged in as admin generates file with data for all enterprises
     Failure/Error: sleep 0.1 until downloaded?
     
     Timeout::Error:
       execution expired
     
     [Screenshot Image]: /home/ffurtado/openfoodnetwork/tmp/capybara/screenshots/failures_r_spec_example_groups_enterprise_fee_summaries_csv_downloads_smoke_test_for_generation_of_report_based_on_permissions_when_logged_in_as_admin_generates_file_with_data_for_all_enterprises_830.png

     
     # ./spec/support/downloads_helper.rb:35:in `sleep'
     # ./spec/support/downloads_helper.rb:35:in `block in wait_for_download'
     # ./spec/support/downloads_helper.rb:34:in `wait_for_download'
     # ./spec/support/downloads_helper.rb:12:in `downloaded_filename'
     # ./spec/system/admin/reports/enterprise_fee_summaries_spec.rb:99:in `block (5 levels) in <top (required)>'
     # ./spec/system/support/cuprite_setup.rb:39:in `block (2 levels) in <main>'

Finished in 17.06 seconds (files took 4.83 seconds to load)
1 example, 1 failure
```